.github/workflows/cookie-management-test.yml

```
@@ -0,0 +1,86 @@
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build Browser Bundle
+        run: npm run build:browser
+
+      - name: Test Browser Bundle Size
+        run: |
+          BUNDLE_SIZE=$(stat -c%s "dist/browser/footer.bundle.js")
+          echo "Browser bundle size: $BUNDLE_SIZE bytes"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)